CHANGELOG.md

```diff
@@ -2,6 +2,78 @@
+- **Maintenance-Gems:** Bump rubocop-rails from 2.23.1 to 2.25.1 [\#5598](https://github.com/consuldemocracy/consuldemocracy/pull/5598)
+- **Maintenance-Gems:** Bump rubocop-performance from 1.20.2 to 1.21.1 [\#5596](https://github.com/consuldemocracy/consuldemocracy/pull/5596)
+- **Maintenance-Gems:** Bump rubocop-capybara from 2.20.0 to 2.21.0 [\#5595](https://github.com/consuldemocracy/consuldemocracy/pull/5595)
+- **Maintenance-Gems:** Bump rspec-rails from 6.1.2 to 6.1.3 [\#5594](https://github.com/consuldemocracy/consuldemocracy/pull/5594)
+- **Maintenance-Gems:** Bump rubocop-rspec from 2.27.0 to 3.0.1 [\#5579](https://github.com/consuldemocracy/consuldemocracy/pull/5579)
+- **Maintenance-Gems:** Bump letter_opener_web from 2.0.0 to 3.0.0 [\#5566](https://github.com/consuldemocracy/consuldemocracy/pull/5566)
+- **Maintenance-Gems:** Bump faker from 3.3.1 to 3.4.1 [\#5565](https://github.com/consuldemocracy/consuldemocracy/pull/5565)
+- **Maintenance-Gems:** Bump rubocop from 1.61.0 to 1.64.1 [\#5564](https://github.com/consuldemocracy/consuldemocracy/pull/5564)
+- **Maintenance-Gems:** Bump recipient_interceptor from 0.3.1 to 0.3.2 [\#5559](https://github.com/consuldemocracy/consuldemocracy/pull/5559)
+- **Maintenance-Gems:** Bump spring from 4.1.3 to 4.2.1 [\#5527](https://github.com/consuldemocracy/consuldemocracy/pull/5527)
+- **Maintenance-Deployment:** Upgrade Ruby to version 3.2.5 [\#5651](https://github.com/consuldemocracy/consuldemocracy/pull/5651)
+- **Maintenance-JavaScript:** \[Security\] Bump micromatch from 4.0.7 to 4.0.8 [\#5661](https://github.com/consuldemocracy/consuldemocracy/pull/5661)
+- **Maintenance-JavaScript:** Bump stylelint-scss from 6.2.1 to 6.3.2 [\#5592](https://github.com/consuldemocracy/consuldemocracy/pull/5592)
+- **Maintenance-JavaScript:** Bump stylelint from 16.3.1 to 16.6.1 [\#5550](https://github.com/consuldemocracy/consuldemocracy/pull/5550)
+- **Maintenance-JavaScript:** Bump @stylistic/stylelint-plugin from 2.1.1 to 2.1.2 [\#5518](https://github.com/consuldemocracy/consuldemocracy/pull/5518)
+- **Maintenance-Refactoring:** Use Rails 7.0 methods to simplify code [\#5491](https://github.com/consuldemocracy/consuldemocracy/pull/5491)
+
+### Fixed
+
+- **Bug:** Don't overwrite node_modules with Docker Compose [\#5690](https://github.com/consuldemocracy/consuldemocracy/pull/5690)
+- **Bug:** Exclude irrelevant Consul Democracy tests in knapsack [\#5687](https://github.com/consuldemocracy/consuldemocracy/pull/5687)
+- **Bug:** Use bundle exec to run delayed jobs on reboot [\#5686](https://github.com/consuldemocracy/consuldemocracy/pull/5686)
+- **Bug:** Fix custom locales not taking precedence [\#5660](https://github.com/consuldemocracy/consuldemocracy/pull/5660)
+- **Bug:** Make sure we only return public records in the API [\#5637](https://github.com/consuldemocracy/consuldemocracy/pull/5637)
```

Comment:
We usually don't say "Bug" under "Fixed" because it's redundant :thinking:.

```suggestion
- **Maintenance-Docker:** Don't overwrite node_modules with Docker Compose [\#5690](https://github.com/consuldemocracy/consuldemocracy/pull/5690)
- **Maintenance-CI:** Exclude irrelevant Consul Democracy tests in knapsack [\#5687](https://github.com/consuldemocracy/consuldemocracy/pull/5687)
- **Maintenance:** Use bundle exec to run delayed jobs on reboot [\#5686](https://github.com/consuldemocracy/consuldemocracy/pull/5686)
- **Translations:** Fix custom locales not taking precedence [\#5660](https://github.com/consuldemocracy/consuldemocracy/pull/5660)
- **GraphQL:** Make sure we only return public records in the API [\#5637](https://github.com/consuldemocracy/consuldemocracy/pull/5637)
```